data/ka'het/ka'het missions.txt

```diff
@@ -293,3 +293,411 @@ mission "Ringworld Debris: Quarg"
+				goto dontcare
+			label planets
+			`	"Oh I have, many times. I have never been on a Builder station, but on a few occasions I have participated to expeditions to newly discovered cities or semi-submersed bunkers. I presume Taely already told you about the troubled beginnings we had in exploring the area; as a consequence of that, we had to reevaluate the procedures for exploring the nearby systems and species, especially beyond the Ossipago wormhole. Every scout ship entering the area has to always remain cloaked for safety; this means that exploring planets is only possible for a limited time and that you can't leave your ship if you don't have a mean of hiding yourself from them.`
+			`	"Because of this, most of the planetary exploration is done with robotic probes. The Ka'het are able to detect and destroy devices that use faster than light communication, so only remote control with a Starling in the same system is possible. There are a lot of compromises that must be taken, but we can still make considerable progress even with these restrictions.`
+			label dontcare
+			`	"I need to prepare a few more instruments before we land," Dusk says as he stops staring at the dome and returns to his equipment. As you reach the complex, you do a few flights around it in search for a good landing site. Dusk indicates to you a spot right next to a set of small circular indents on the structure. After touching down, you and Dusk don extravehicular suits and start walking towards the cupola. He stops in front of one ring and starts looking through his equipment.`
+			choice
+				`	"What is that?"`
+			`	"Well, I obviously cannot be sure, but given its location and that on the other side there is a pressurized area, it should be an airlock. I have seen scans of the Builder stations, and they look quite similar." He takes a heavy-looking scanner, puts it on the wall of the structure and starts moving it around. "If I am right, this way should have a control station hidden inside the wall - and here it is!" he sings with satisfaction.`
+			`	Using a small instrument similar in appearance to a crowbar, Dusk slowly pulls out a section from the wall, kicking off a good amount of dust in the process. The control station is a two sided panel, full of switches and buttons with small alien markings above each of them.`
+			choice
+				`	"Can you read them?"`
+				`	"Can you open the airlock?"`
+					goto open
+				`	"Are you sure this won't put us in danger?"`
+					goto danger
+			`	"I can, yes. The first row of switches controls the lights inside the complex, the second row controls redundant pressurization systems, and there are similar such trivial operations controlled by the rest of the panel.`
```

Comment:
```suggestion
			`	"I can, yes. The first row of switches controls the lights inside the complex, the second row controls pressurization systems, and there are similar such operations controlled by the rest of the panel.`
```
If these switches let people into the station, are they really redundant or trivial? Those words usually have a negative connotation and mean something that's useless or unnecessary.

Reply:
Hm I agree that trivial should be removed, but redundant here is in its more technical meaning, as in "something that replaces the function of something else in case it fails". E.g., Shuttle has many redundant system to prevent loss of control of the vehicle," "lunar starship features two independent airlocks for additional redundancy," etc

Reply:
Gotcha. Here's an updated suggestion then.
```suggestion
			`	"I can, yes. The first row of switches controls the lights inside the complex, the second row controls redundant pressurization systems, and there are similar such operations controlled by the rest of the panel.`
```